### PR TITLE
test for duplicate bookmark name

### PR DESCRIPTION
**Pull request by peterdragun**
Dependency: https://github.com/SatelliteQE/airgun/pull/603
test coverage for 1920566

Test results:
```pytest -v tests/foreman/ui/test_bookmarks.py -k test_negative_create_with_duplicate_name
============================================================================== test session starts ==============================================================================
platform linux -- Python 3.9.5, pytest-6.2.4, py-1.10.0, pluggy-0.13.1 -- /home/pdragun/.virtualenvs/robottelo3.9/bin/python
cachedir: .pytest_cache
shared_function enabled - OFF - scope:  - storage: file
rootdir: /home/pdragun/Documents/robottelo, configfile: pyproject.toml
plugins: ibutsu-1.16, xdist-2.3.0, forked-1.3.0, mock-3.6.1, reportportal-5.0.8, services-2.2.1, cov-2.12.1
collected 5 items / 4 deselected / 1 selected                                                                                                                                   

tests/foreman/ui/test_bookmarks.py::test_negative_create_with_duplicate_name PASSED                                                                                       [100%]

============================================================ 1 passed, 4 deselected, 0 warnings in 102.34s (0:01:42) ============================================================```

### CODE REVIEW

**Comment by omkarkhatavkar** [comment]
Can one of the admins verify this patch?

File: tests/foreman/ui/test_bookmarks.py [hunk 1]
```diff
@@ -252,3 +253,34 @@ def test_negative_delete_bookmark(random_entity, default_viewer_role, test_name)
...
+
+    :Setup:
+
+        1. Create a bookmark of a random name with random query
+
+    :Steps:
+
+        1. Create new bookmark with duplicate name.
+
+    :expectedresults: Bookmar can't be created, submit button is disabled
```
Comment (swadeley) [inline tests/foreman/ui/test_bookmarks.py:272]:
```suggestion
    :expectedresults: Bookmark can't be created, submit button is disabled
```

File: tests/foreman/ui/test_bookmarks.py [hunk 2]
```diff
@@ -252,3 +253,34 @@ def test_negative_delete_bookmark(random_entity, default_viewer_role, test_name)
...
+
+@pytest.mark.tier2
+def test_negative_create_with_duplicate_name(session, random_entity):
+    """Create bookmark with duplicate name
+
+    :id: 18168c9c-bdd1-4839-a506-cf9b06c4ab44
+
+    :Setup:
+
+        1. Create a bookmark of a random name with random query
```
Comment (swadeley) [inline tests/foreman/ui/test_bookmarks.py:266]:
```suggestion
        1. Create a bookmark of a random name with random query.
```

**Review by swadeley — APPROVED** [review]
ACK

**Comment by swadeley** [comment]
Its nice to have the full BZ name and link in the description:
[Bug 1920566 - Saving bookmark with an existing/duplicate bookmark name doesn't show a clear error](https://bugzilla.redhat.com/show_bug.cgi?id=1920566)

**Comment by mshriver** [comment]
Not seeing the robottelo-runner check get added, but the test passed in CI.

**Pull request merged**